src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Cars {
+    private static final List<Car> cars = new ArrayList<>();
+
+    private Cars() {
+    }
+
+    public static void setCarsByName(List<String> names) {
+        validateNonDuplicatedNames(names);
+        for (String name : names) {
+            cars.add(new Car(name));
+        }
+    }
+
+    public static List<Car> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public static int getMaxPositionByCars() {
+        return cars.stream()
+                .max(Comparator.comparingInt(Car::getPosition))
+                .get().getPosition();
```

Comment:
Car 의 포지션을 비교 한 후 poistion을 갖고와도 되긴 하는데 아래처럼 먼저 Position들을 뽑아내서 Max값을 뽑을 수도 있어요 😁 

```suggestion
        return cars.stream()
                .map(Car::getPosition)
                .max(Integer::compareTo)
                .orElse(0);
```